commit 02d5287d82b2bd583f14f2d29be41ce5796667f5
Author: Jason Gross <jgross@mit.edu>
Date:   Thu Sep 7 20:23:29 2023 -0700

    Support coq-native by passing through `COQEXTRAFLAGS`
    
    Since coq-interval doesn't build nor install .coq-native files, when
    coq-native is installed, vcfloat should avoid looking for .coq-native
    files.


diff --git a/vcfloat/Makefile b/vcfloat/Makefile
index 062b6bc..78fad69 100644
--- a/vcfloat/Makefile
+++ b/vcfloat/Makefile
@@ -1,11 +1,12 @@
 # This Makefile assumes that coq already has access to
 # Flocq, Interval, CompCert, etc.,
-# which will be true if coq was installed through opam 
+# which will be true if coq was installed through opam
 # or as a "Coq Platform" package.
 
 COQC=coqc
 COQDEP=coqdep
 COQFLAGS := $(shell cat _CoqProject)
+COQEXTRAFLAGS?=
 
 target:  vcfloat2 tests
 
@@ -16,7 +17,7 @@ tests: ../Test/Test.vo ../Test/TestFunc.vo ../Test/TestPaper.vo ../Test/Test2.vo
 vcfloat2:  VCFloat.vo FPCompCert.vo Version.vo FPLib.vo
 
 %.vo: %.v
-	$(COQC) $(COQFLAGS) $*.v
+	$(COQC) $(COQFLAGS) $(COQEXTRAFLAGS) $*.v
 
 ifdef INSTALLDIR
 INSTALLFILES=$(shell make -Bn vcfloat2 2>/dev/null | awk '/^coqc.*v$$/{print $$NF"o"}')
@@ -31,7 +32,7 @@ endif
 	install -d $(INSTALLDIR)
 	install -m 0644 $(INSTALLFILES) $(INSTALLDIR)
 
-depend: 
+depend:
 	$(COQDEP) $(COQFLAGS) *.v ../Test/*.v > .depend
 
 ifneq ($(MAKECMDGOALS),depend)
